core/pom.xml

```diff
@@ -51,10 +51,10 @@
 
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-jdk14</artifactId>
+            <artifactId>slf4j-api</artifactId>
             <!-- Should be the same version as HikariCP uses -->
             <version>1.7.25</version>
-            <scope>runtime</scope>
+            <scope>provided</scope>
```

Comment:
Do make sure that slf4j is still shaded on bungee and bukkit

Reply:
It is shaded in the bundle module